### PR TITLE
LPS-186313 [PRM] [BUG] Fix Buttons with wrong links

### PR DESCRIPTION
**Pull request by DiogoNo**
Ticket: https://issues.liferay.com/browse/LPS-186313

### CODE REVIEW

**Comment by thaynam** [comment]
ci:test:sf

**Comment by thaynam** [comment]
ci:test:solutions-acceptance

**Comment by thaynam** [comment]
ci:test:relevant

**Comment by thaynam** [comment]
ci:test:stable

**Review by AfonsoHSantos — APPROVED** [review]
The test failures (solutions-acceptance) aren't related with this PR

**Comment by jairmedeiros** [comment]
ci:forward

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/02_home/page-definition.json [hunk 1]
```diff
@@ -190,6 +190,24 @@
 											"id": "link",
 											"value": {
 												"fragmentLink": {
+													"href": {
+														"value_i18n": {
+															"ar_SA": "https://partner.liferay.com/documents/72519/1450806/Liferay+Solution+Partner+Program+Guide+2023.pdf/22a6eca6-1b1b-6f37-9b0c-441ba069ef6b?t=1671150824668",
```
Comment (jairmedeiros) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/src/main/resources/site-initializer/layouts/02_home/page-definition.json:195]: It need to use a site-initializer replace

**Review by jairmedeiros — CHANGES_REQUESTED** [review]

**Comment by thaynam** [comment]
[resent](https://github.com/pt-liferay-solutions/liferay-portal/pull/4255)

**Pull request closed without merging**